k-distribution/include/kframework/ktest-kdep.mak

```diff
@@ -14,4 +14,4 @@ all: $(TESTS)
 dummy:
 
 %.k %.md: dummy
-	$(KDEP) $(OPTS) $@ | sed 's!'`pwd`'/\(\./\)\{0,2\}!!g' $(CHECK) $@.out
+	$(KDEP) $(OPTS) $@ | sed 's!'`pwd`'/\(\./\)\{0,2\}!!g' | sort $(CHECK) $@.out
```

Comment:
I've addressed these comments - as regards 2/3, I think the best way is to use a new `.mak` as Erik implemented it originally. The `ktest.mak` version does a lot of extra work (rather than just invoking kdep, which doesn't even need to `kompile` the source files). We already do this for `keq` tests as well.